aten/src/ATen/native/Convolution.cpp

```diff
@@ -207,6 +218,43 @@ static void check_input_shape_forward(const at::Tensor& input,
              "Given weight of size ", weight.sizes(),
              ", expected bias to be 1-dimensional with ", weight.size(0), " elements",
              ", but got bias of size ", bias.sizes(), " instead");
+
+    for (int i = 2; i < k; ++i) {
+      input_shape.push_back(input.size(i) + 2 * padding[i-2]);
+      kernel_shape.push_back(weight.size(i));
+      if (input_shape.back() < kernel_shape.back()) {
+        kernel_size_correct = false;
+      }
+
+      // TODO: Should output padding be considered while computing output_shape?
```

Comment:
output_padding is only used in transposed convolutions

Reply:
dropping comment